implementations/rust/ockam/ockam_command/src/terminal/tui.rs

```diff
@@ -14,7 +16,21 @@ pub trait ShowCommandTui {
     async fn get_arg_item_name_or_default(&self) -> miette::Result<String>;
     async fn list_items_names(&self) -> miette::Result<Vec<String>>;
     async fn show_single(&self, item_name: &str) -> miette::Result<()>;
-    async fn show_multiple(&self, items_names: Vec<String>) -> miette::Result<()>;
+    async fn show_multiple(&self, items_names: Vec<String>) -> miette::Result<()> {
+        for item_name in items_names {
+            if self.show_single(&item_name).await.is_err() {
+                self.terminal()
+                    .stdout()
+                    .plain(fmt_warn!(
+                        "Failed to show {} {}",
+                        Self::ITEM_NAME.singular(),
+                        color_primary(item_name)
+                    ))
+                    .write_line()?;
+            }
+        }
+        Ok(())
+    }
```

Comment:
For a separate PR:

I'd propose a different approach here. Since we want to show multiple items with the same structure as when showing a single item, we should change [this line](https://github.com/build-trust/ockam/blob/04cf73ab070197a6502e547da66e26adbe063494/implementations/rust/ockam/ockam_command/src/terminal/tui.rs#L77) to iterate the list and call `show_single`.

Then, remove the `show_multiple` method and add a new one `show_list()` that internally calls the `list_items_names` + `build_list`. That way we can reuse the implemented trait for both `show` and `list` commands.

Reply:
Lists are supposed to be a less verbose version compared to the single element. If we always assume the same structure we lose this aspect.
Maybe we can have a `show_single(verbose: bool)`?